.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build documentation
+    if: ${{ secrets.ENCRYPTION_KEY }}
+    strategy:
+      matrix:
+        scala: [ 2.13.3, 2.12.12 ]
+        java: [ openjdk@1.8, openjdk@1.11 ]
```

Comment:
since this is just docs we can just run against the default scalaVersion and just use jdk 11